crates/cxx-qt-build/src/out.rs

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! A module for handling anything output related
+
+#[cfg(windows)]
+use std::path::Path;
+
+#[cfg(windows)]
+pub(crate) fn write_file(
+    path: impl AsRef<Path>,
+    contents: impl AsRef<[u8]>,
+) -> std::io::Result<()> {
+    // On Windows we're experiencing some issues with file locking and possibly
+    // synchronization, especially when combined with cc::Build
+    // So force a synchronization here with sync_all()
+    let file = File::create(path)?;
```

Comment:
need to add a `use` for this :-)

Reply:
Done, fixed some other issues as well by disabling the `#[cfg(windows)]` temporarily.
It's amazing how many mistakes you can make when coding blind :sweat_smile: